cocotb/share/makefiles/Makefile.rules

```diff
@@ -50,14 +50,18 @@ SIM_DEFINE = IUS
 endif
 
 # Add Simulator Define for compilation
-GCC_ARGS+= -D$(SIM_DEFINE)
-GXX_ARGS+= -D$(SIM_DEFINE)
+C_ARGS+= -D$(SIM_DEFINE)
+CXX_ARGS+= -D$(SIM_DEFINE)
```

Comment:
nit: indentation, and please add space before `+=`

Reply:
meta-nit: this is alignment, not indentation